Common/Inc/GlobalFlag.hpp

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Is there a reason LaminarOS was updated? is this needed?

Reply:
Accidental, will try to undo